src/project/qml/MuseScore/Project/internal/NewScore/KeySignatureSettings.qml

```diff
@@ -97,6 +97,7 @@ FlatButton {
 
                 StyledTabButton {
                     id: majorTab
+                    //: mode of a key signature, not an interval
                     text: qsTrc("project/newscore", "Major")
```

Comment:
Might as well add the disambiguation here too. We should probably get in the habit of doing this everywhere we use "Major" and "Minor".

Reply:
OK, there are only those 2 time 3, as far as I can tell